src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.netstdref.cs

```diff
@@ -0,0 +1,53 @@
+﻿#pragma warning disable 108 // new keyword hiding
+#pragma warning disable 114 // new keyword hiding
+using System;
+using System.Collections.Generic;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+
+namespace Windows.UI.Xaml.Controls
+{
+	public partial class ListViewBase
+	{
+		//private int PageSize => throw new NotImplementedException();
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)